### ISSUE TITLE
OpenSSL dependency seems broken upon YunoHost installation

### ISSUE THREAD

**Issue by tituspijean** [YunoHost/issues#2261]
### Describe the bug

cf. forum thread https://forum.yunohost.org/t/rasperry-pi-2b-yunohost-installation-failing-due-to-openssl-package-conflict/26621
for RPi 2 and enclosed logs for RPi 0

> The following packages have unmet dependencies:
Yunohost : Conflicts: openssl (>= 1.1.1o-0) but 1.1.1w-0~deb11u1+rpt1 is to be installed.

### Context

- Hardware: RPI 0 and 2
- YunoHost version: 11.2

### Logs

[yunohost-installation_20231004_223059.log](https://github.com/YunoHost/issues/files/12810014/yunohost-installation_20231004_223059.log)

**Comment by OniriCorpe** [comment]
also on my x86_64 server:

```bash
# apt upgrade openssl
Les paquets suivants contiennent des dépendances non satisfaites :
 yunohost : Est en conflit avec: openssl (>= 1.1.1o-0) mais 1.1.1w-0+deb11u1 devra être installé
E: Paquets défectueux
```

**Comment by OniriCorpe** [comment]
> also on my x86_64 server:

fixed for my (already installed) server with the ynh [v11.2.5](https://github.com/YunoHost/yunohost/blob/dev/debian/changelog)

**Comment by alexAubin** [comment]
Yup, fixed in 11.2.5
